project/Dependencies.scala

```diff
@@ -13,7 +13,7 @@ object Dependencies {
   val nailgunCommit = "a2520c1e"
 
   val zincVersion = "1.3.0-M4+32-b1accb96"
-  val bspVersion = "2.0.0-M5"
+  val bspVersion = "2.0.0-M5+1-e75ba0aa" // TODO bump bsp to make it compile
```

Comment:
The release is out now
```suggestion
  val bspVersion = "2.0.0-M7" // TODO bump bsp to make it compile
```